artemis-server/src/main/java/org/apache/activemq/artemis/core/config/Configuration.java

```diff
@@ -357,6 +357,7 @@
     * @param uri  the URI of the acceptor
     * @return this
     * @throws Exception in case of Parsing errors on the URI
+    * @see <a href="https://activemq.apache.org/artemis/docs/latest/configuring-transports.html">Configuring the Transport</a>
```

Comment:
IMO This isnt a good idea, as location of docs url can change in time, and once code is tagged and release we cannot go back and change the links in previously released code.

Reply:
I will usee the github address instead.